airflow/utils/db.py

```diff
@@ -593,6 +593,14 @@ def create_default_connections(session: Session = NEW_SESSION):
         ),
         session,
     )
+    merge_conn(
+        Connection(
+            conn_id="tabular_default",
+            conn_type="tabular",
+            extra='{"extra__tabular__signerUrl": "https://api.tabulardata.io/ws/v1"}'
```

Comment:
In the form widgets we have `extra__tabular__baseUrl`. Is this an additional user enterable field that we should add to the form widget as well?

Reply:
Good catch! I switched that around a couple of times :)